x-pack/plugins/security_solution/server/lib/detection_engine/signals/mappings/build_severity_from_mapping.test.ts

```diff
@@ -22,5 +22,45 @@ describe('buildSeverityFromMapping', () => {
     expect(severity).toEqual({ severity: 'low', severityMeta: {} });
   });
 
+  test('severity is overridden to highest matched mapping', () => {
+    const severity = buildSeverityFromMapping({
+      doc: sampleDocSeverity(23),
+      severity: 'low',
+      severityMapping: [
+        { field: 'event.severity', operator: 'equals', value: '23', severity: 'critical' },
+        { field: 'event.severity', operator: 'equals', value: '23', severity: 'low' },
+        { field: 'event.severity', operator: 'equals', value: '11', severity: 'critical' },
+        { field: 'event.severity', operator: 'equals', value: '23', severity: 'medium' },
+      ],
+    });
+
+    expect(severity).toEqual({
+      severity: 'critical',
+      severityMeta: {
+        severityOverrideField: 'event.severity',
+      },
+    });
+  });
+
+  test('severity is overridden when field is event.severity and source value is number', () => {
+    const severity = buildSeverityFromMapping({
+      doc: sampleDocSeverity(23),
+      severity: 'low',
+      severityMapping: [
+        { field: 'event.severity', operator: 'equals', value: '13', severity: 'loew' },
```

Comment:
You want `loew` instead of `low` here?

Reply:
This is typed out to be `SeverityMappingOrUndefined` -- any idea why the type-checker didn't catch this?